tce/src/applibs/Scheduler/ProgramRepresentations/CFG/BasicBlockNode.hh

```diff
@@ -102,6 +102,16 @@ public:
     void setLoopScheduled() { loopScheduled_ = true; }
     void setBBOwnership(bool ownership = true) { bbOwned_ = ownership; }
 
+    // hwloop flags
+    bool
+    isHWLoop() const {
```

Comment:
Please add a more thorough comment describing what are the conditions when this is set to true.